app/controllers/wikis_controller.rb

```diff
@@ -48,19 +49,22 @@ def create
   end
 
   def edit
-     @user = current_user
+     #@user = current_user
      #@user = User.find(params[:user_id])
-     @wikis = @user.wikis
-     @wiki = @wikis.find(params[:id])
-     #render edit_user_wiki_path
+     #@wikis = @user.wikis
+     #@wiki = @wikis.find(params[:id])
+     @wikis = Wiki.all
+     @wiki = Wiki.find(params[:id])
   end
   
   def update
-    @user = current_user
+    #@user = current_user
     #@user = User.find(params[:id])
-    @wikis = @user.wikis
-    @wiki = @wikis.find(params[:id])
+    #@wikis = @user.wikis
+    #@wiki = @wikis.find(params[:id])
     #p @wiki
+    @wikis = Wiki.all
+    @wiki = @wikis.find(params[:id])
     @wiki.title = params[:wiki][:title]
```

Comment:
To use pundit in the `update` action, add this line to `update`:

```diff
...
     @wiki = @wikis.find(params[:id])
+    authorize @wiki
     @wiki.title = params[:wiki][:title]
    end
```
This new line will call the `WikiPolicy#update?` method. If the current user is not authorized to update the wiki, a `Pundit::NotAuthorizedError` will be raised.